pallets/gear/src/benchmarking/code.rs

```diff
@@ -71,6 +71,9 @@ pub struct ModuleDefinition {
     /// Function body of the exported `handle` function. Body is empty if `None`.
     /// Its index is `imported_functions.len() + 1`.
     pub handle_body: Option<FuncBody>,
+    /// Function body of the exported `handle_reply` function. Body is missing if `None`.
+    /// Its index is `imported_functions.len() + 2`.
+    pub reply_body: Option<FuncBody>,
     /// Function body of a non-exported function with index `imported_functions.len() + 2`.
```

Comment:
it's also +2

Reply:
change this https://github.com/gear-tech/gear/blob/c8b553e22d2a8ced1d297a6b8a503847ae1505ac/pallets/gear/src/benchmarking/mod.rs#L1903

Reply:
not only there
everywhere where "aux" appears: not only TableSegment, but Call(num)'s also